style.css

```diff
@@ -0,0 +1,67 @@
+body {
+    font-size: 1.6rem;
+    line-height: 1.5;
+    text-align: center;
+    color: #333;
+    margin: 0;
+}
+
```

Comment:
well done on mentioning the use of linters , but I suggest that you put this comment in the final review message since they didn't add any linter files , inline comment are added next to a line or block of code and it's content should be in that context otherwise it becomes confusing .